test/models/user_test.rb

```diff
@@ -0,0 +1,87 @@
+require "test_helper"
+
+describe User do
+
+  # relation tests: a user has many votes
+  describe 'has relations' do
+    it 'a user can have votes' do
+      user = users.first
+      work_first = works.first
+      work_second = works.second
+
+      vote = Vote.create!(user_id: user.id, work_id: work_first.id)
+      vote = Vote.create!(user_id: user.id, work_id: work_second.id)
```

Comment:
instead of reassigning the same variable `vote`, when all you really care about is making the Votes and not having a reference to them by some variable, you should feel free to write this instead:
```ruby
Vote.create!(user_id: user.id, work_id: work_first.id)
Vote.create!(user_id: user.id, work_id: work_second.id)
```